hibike/display.py

```diff
@@ -1,18 +1,19 @@
 import hibike
 import time
 h = hibike.Hibike(timeout=1)
+h.addVirtualPort('/dev/pts/26')
```

Comment:
Is there any way to avoid hardcoding a port here? (Maybe accept the port via a command line arg?)